probe_scraper/exc.py

```diff
@@ -1,4 +1,20 @@
-class ProbeScraperInvalidRequest(Exception):
+class ProbeScraperError(Exception):
+    """Exception type for returning errors in push mode."""
+
+    def __init__(self, message, status_code):
+        self.status_code = status_code
+        self.message = message
```

Comment:
Up to you, but I like having these in the same order as the args list.

Reply:
```suggestion
        self.message = message
        self.status_code = status_code
```